### ISSUE TITLE
Alternatives? (PROJECT DEAD)

### ISSUE THREAD

**Issue by vincenzogianfelice** [RoganDawes/P4wnP1_aloa#320]
By now this project is dead, (I also encountered compile errors on raspberry OS). Is there any other alternative?

**Comment by nototter** [comment]
yes, there is an alternative, plus you can still use your waveshare oled

[nototter/artemis](https://github.com/nototter/artemis)
its in python, which isn't very fast, but it works
it also has nearly the same features; hid, usb exfil, emulation, etc. and wifi support
more features will be added eventually, but until then it works the same
